spark/src/main/scala/geotrellis/spark/io/s3/S3AttributeStore.scala

```diff
@@ -33,6 +33,9 @@ class S3AttributeStore(bucket: String, rootPath: String) extends AttributeStore[
 
   def path(parts: String*) = parts.filter(_.nonEmpty).mkString("/")
 
+  def attributePath(id: LayerId): String =
+    path(rootPath, "_attributes", s"*__${id.name}__${id.zoom}.json")
```

Comment:
Yes, I see; so how to solve it, to change attribute storage and to make it depend on layerId? Or to search for some workaround?